File: tools/content/__fixtures__/content/ok/tracks/dsa/problems/lc-0001-two-sum/solution.py
from typing import Dict, List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # index of every value seen so far
        seen: Dict[int, int] = {}
        for index, value in enumerate(nums):
            if target - value in seen:
                return [seen[target - value], index]
            seen[value] = index
        return []

File: tools/content/__fixtures__/content/ok/tracks/dsa/problems/lc-0001-two-sum/solution.go
package main

func twoSum(nums []int, target int) []int {
	seen := map[int]int{}
	for i, n := range nums {
		if j, ok := seen[target-n]; ok {
			return []int{j, i}
		}
		seen[n] = i
	}
	return []int{}
}
